core/templates/dev/head/css/oppia.css

```diff
@@ -77,6 +81,10 @@ button.md-button.md-default-theme.oppia-learner-continue-button:hover {
 .material-icons.md-dark {
   color: rgba(0, 0, 0, 0.54);
 }
+.middle{
```

Comment:
![untitled](https://cloud.githubusercontent.com/assets/8307266/16185233/ea372efc-36df-11e6-87b0-9f18c8d14acc.png)

@seanlip  @souravbadami 
This is what it looks like when we use ``` md-18 ```
Browser : Firefox 47.0
Platform : Windows 8.1
Thanks